microscopy_SEM001/README

```diff
@@ -0,0 +1,7 @@
+Part of SEM dataset for AxonDeepSeg (https://axondeepseg.readthedocs.io/)
+
+1 rat spinal cord sample with axon and myelin manual segmentation labels.
+
+Example empty dataset containing scanning electron microscopy (SEM) data to illustrate BIDS convention.
+
+BIDS version 1.6.0 - Microscopy BEP031 version 0.0.5 (2021-07-22T14:22:00)
```

Comment:
```suggestion
1 mouse spinal cord sample imaged over 2 sessions.

Example dataset with empty images containing scanning electron microscopy (SEM) data to illustrate BIDS convention.

```

Thisis just a suggestion but maybe we should not mention ADS because here there is no data left, only partial metadata.
In addition, we say "rat" spinal cord but in participants.tsv, it's a mouse! :)
I would also add a `ses-02`, see comment in `sub-01_sessions.tsv`.
The BIDS version is not necessary in the readme.